tests/psalm/IntMaskOfTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Typhoon\Type\IntMaskOfTest;
+
+use Typhoon\Type\IntMaskOfType;
+use Typhoon\Type\Type;
+use function Typhoon\Type\extractType;
+
+/**
+ * @param Type<\ReflectionClass::IS_*> $constantType
+ */
+function a(Type $constantType): void
```

Comment:
It would be great to add one more test for `IntMaskType`.

Reply:
Done!